components/public-api/gitpod/experimental/v1/user.proto

```diff
@@ -52,24 +53,23 @@ service UserService {
 
     rpc GetGitToken(GetGitTokenRequest) returns (GetGitTokenResponse) {}
 
-    rpc BlockUser(BlockUserRequest) returns (BlockUserResponse) {}
-}
+    // GetSuggestedRepoURLs returns a list of suggested repositories to open for
+    // the user.
+    rpc GetSuggestedRepoURLs(GetSuggestedRepoURLsRequest)
+        returns (GetSuggestedRepoURLsResponse) {}
```

Comment:
I don't think this belongs into user-service. We should introduce an `scm-service` for this. We can move things like getGitToken, resolveContextURL, etc. to it as well (not necessarily this PR 😇 ).

Reply:
good catch, thanks!

Reply:
resolved in https://github.com/gitpod-io/gitpod/pull/18514/commits/5bf40badb49b4c1b1581e5cdf854aa0ad6ed57c8

Reply:
I'll leave the API migration for another PR - maybe there we can figure out if we need to keep the user APIs as deprecated or if it's safe to just remove them because our clients are not using it